src/TaskParameters.ts

```diff
@@ -52,46 +47,50 @@ export default class TaskParameters {
 
     constructor() {
         var locations = ["eastus", "eastus2", "westcentralus", "westus", "westus2", "northeurope", "westeurope"];
-        // general inputs
+
         console.log("start reading task parameters...");
-        this.location = tl.getInput(constants.Location, { required: true });
 
-        if (!(locations.indexOf(this.location.toString().replace(/\s/g, "").toLowerCase()) > -1)) {
-            throw new Error("location not from available regions or it is not defined");
-        }
-        this.resourceGroupName = tl.getInput(constants.ResourceGroupName, { required: true });
-        this.managedIdentity = tl.getInput(constants.ManagedIdentity, { required: true });
         this.imagebuilderTemplateName = tl.getInput(constants.ImageBuilderTemplateName);
         if (this.imagebuilderTemplateName.indexOf("json") > -1) {
```

Comment:
It should be endsWith and that too with a ".json" instead of just "json"

Reply:
done